static/js/composebox_typeahead.js

```diff
@@ -392,41 +392,49 @@ function should_show_custom_query(query, items) {
 }
 
 export const slash_commands = [
-    {
-        text: $t({defaultMessage: "/dark (Toggle night mode)"}),
-        name: "dark",
-    },
     {
         text: $t({defaultMessage: "/day (Toggle day mode)"}),
-        name: "day",
+        name: "day", // first keyword to match
+        name2: "light", // second keyword to match
+        match: ["name", "name2"], // list of attributes to match
```

Comment:
I think perhaps a cleaner interface might be to have a `name` primary value and an `aliases` list of alternative names?  That's what other databases with this architecture often use.

Reply:
Thanks for the feedback! I've implemented the suggested changes in my latest push.